src/Checkbox/CheckboxLabel.js

```diff
@@ -4,6 +4,8 @@ import Flex from '../Flex';
 
 const CheckboxLabel = styled(Flex.withComponent('label'))`
   cursor: ${props => (props.disabled ? 'not-allowed' : 'pointer')};
+  align-items: start;
```

Comment:
Looks like the reason we couldn't use a default prop here is because in `src/Checkbox/index` we're setting the align-items prop on the label. Let's just update it there to start.